### DISCUSSION TITLE
moving from v4 to v5

### DISCUSSION THREAD

**Discussion by vespino** [chillerlan/php-qrcode#263] [Q&A] [answered]
Apparently I've been using version 4 of this awesome script for too long and since I've just started using a new server, composer installed version 5 for me. Everything still works fine, although it's now generating very big SVG QR codes instead of small 285x285px PNG codes that I'm used to. This is my code:

```
use chillerlan\QRCode\{QRCode, QROptions};
(new QRCode)->render(<data>);
```

What do I need to change to get the simple QR/PNG 285x285px that was generated by version 4?

**Comment by codemasher** [comment, marked as answer]
Hey, the default output has [changed from PNG to SVG](https://github.com/chillerlan/php-qrcode/releases/tag/5.0.0).

In order to keep the PNG output, you will need to pass a `QROptions` instance just as shown in the GD output example:

https://github.com/chillerlan/php-qrcode/blob/0a897ca7235757a6e59e0949157daa5e8f7c680b/examples/image.php#L17-L22

Alternatively, if you want to keep the (arguably fancier) SVG output, you can simply style the SVG (or containing element) via CSS.
